examples/ios_app/Example/BUILD

```diff
@@ -14,6 +14,7 @@ ios_application(
     app_icons = glob(["Assets.xcassets/AppIcon.appiconset/**"]),
     bundle_id = "io.buildbuddy.example",
     bundle_name = "Example",
+    executable_name = "Examnple_ExecutableName",
```

Comment:
@brentleyjones I am not sure this is working, unless the generated `Info.plist` just has this set.

Reply:
Correct. We need to add support for setting `EXECUTABLE_NAME` based on this value.

Reply:
Ahh, or rules_apple needs to adjust the Info.plist. What does a `bazel buld` produce inside the `.ipa`?

Reply:
Info plist is correct:

```
	<key>CFBundleExecutable</key>
	<string>Example_ExecutableName</string>
```